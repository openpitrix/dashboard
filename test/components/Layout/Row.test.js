import Row from 'components/Layout/Grid/Row';

describe('Layout/Row', () => {
  it('basic render', () => {
    const wrapper = render(
      <Row className="test">
        <div>test</div>
      </Row>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
