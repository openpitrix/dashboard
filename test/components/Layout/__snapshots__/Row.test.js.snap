// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout/Row basic render 1`] = `
<div
  class="row test"
>
  <div>
    test
  </div>
</div>
`;
